src/quoteUtils.js

```diff
@@ -20,7 +22,8 @@ function getQuoteFromConnector (connector, params) {
       .query({
         source_account: params.sourceAccount,
         destination_account: params.destinationAccount,
-        destination_expiry_duration: 2
+        destination_expiry_duration: params.destinationExpiryDuration,
```

Comment:
It does change the sender API (since we allow passing expiries now), but it is backwards compatible. If they aren't passed, the connector just uses the [default](https://github.com/interledger/five-bells-connector/blob/e798484449166f0e0964b0b53a18c7ffc6f82e82/src/models/quote.js#L9).